pynuodb/crypt.py

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Editor cleaned up a lot of whitespace automatically when I saved. Sorry if that makes review more difficult.

Reply:
"Diff settings > Hide whitespace changes"